frontend/public/components/factory/table.tsx

```diff
@@ -415,13 +426,23 @@ type ComponentProps = {
   kindObj?: K8sResourceKindReference;
 };
 
+const getActiveColumns = (Header, componentProps, activeColumns) => {
+  let columns = Header(componentProps);
+  if (!_.isEmpty(activeColumns)) {
+    columns = columns?.filter((col) => activeColumns?.[col.title] || col.title === '');
```

Comment:
I would use something other than `title` to identify the column. We probably want an internal ID for each.

We need to consider how this will work if we translate the UI.

Reply:
Good idea, added an ID.